src/main/java/uk/gov/hmcts/reform/civil/service/GeneralApplicationCreationNotificationService.java

```diff
@@ -114,13 +120,16 @@ private String getLiPTemplate(CaseData civilCaseData, CaseData caseData) {
     private void sendNotificationToGeneralAppRespondent(CaseData caseData, String recipient, String emailTemplate)
         throws NotificationException {
         try {
+            log.info("Sending notification to recipient: {} for Case ID: {} with template: {}", recipient, caseData.getCcdCaseReference(), emailTemplate);
             notificationService.sendMail(
                 recipient,
                 emailTemplate,
                 addProperties(caseData),
                 String.format(REFERENCE_TEMPLATE, caseData.getGeneralAppParentCaseLink().getCaseReference())
             );
+            log.info("Notification sent successfully to recipient: {} for Case ID: {}", recipient, caseData.getCcdCaseReference());
         } catch (NotificationException e) {
+            log.error("Failed to send notification to recipient: {} for Case ID: {}", recipient, caseData.getCcdCaseReference(), e);
```

Comment:
caseData.getCcdCaseReference() is called multiple times here in the same function. Might be best to store it.